src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -233,11 +190,74 @@ ApiGlobe.prototype.addElevationLayersFromJSON = function addElevationLayersFromJ
 ApiGlobe.prototype.addElevationLayersFromJSONArray = function addElevationLayersFromJSONArray(urls) {
     var proms = [];
 
-    for (var i = 0; i < urls.length; i++) {
-        proms.push(Fetcher.json(urls[i]).then(this.addElevationLayer.bind(this)));
+    for (const url of urls) {
+        proms.push(Fetcher.json(url).then(layer => this.addElevationLayer(layer)));
+    }
+
+    return Promise.all(proms);
+};
+
+function updateLayersOrdering(globeLayer, imageryLayers) {
+    var sequence = ImageryLayers.getColorLayersIdOrderedBySequence(imageryLayers);
+
+    var cO = function cO(object) {
+        if (object.changeSequenceLayers)
+            { object.changeSequenceLayers(sequence); }
+    };
+
+    for (const node of globeLayer.level0Nodes) {
+        node.traverse(cO);
+    }
+}
+
+ApiGlobe.prototype.moveLayerUp = function moveLayerUp(layerId) {
+    const imageryLayers = this.scene.getAttachedLayers(l => l.type === 'color');
+    const layer = this.scene.getAttachedLayers(l => l.id == layerId)[0];
+    ImageryLayers.moveLayerUp(layer, imageryLayers);
+    updateLayersOrdering(this.scene._geometryLayers[0], imageryLayers);
+    this.scene.renderScene3D();
+};
+
+ApiGlobe.prototype.moveLayerDown = function moveLayerDown(layerId) {
+    const imageryLayers = this.scene.getAttachedLayers(l => l.type === 'color');
+    const layer = this.scene.getAttachedLayers(l => l.id == layerId)[0];
+    ImageryLayers.moveLayerDown(layer, imageryLayers);
+    updateLayersOrdering(this.scene._geometryLayers[0], imageryLayers);
+    this.scene.renderScene3D();
+};
+
+/**
+ * Moves a specific layer to a specific index in the layer list. This function has no effect if the layer is moved to its current index.
+ * @constructor
+ * @param      {string}  layerId   The layer's idendifiant
+ * @param      {number}  newIndex   The new index
+ */
+ApiGlobe.prototype.moveLayerToIndex = function moveLayerToIndex(layerId, newIndex) {
+    const imageryLayers = this.scene.getAttachedLayers(l => l.type === 'color');
```

Comment:
Yep.
I'll push a commit fixing `getLayerById`, `getLayers` and `getLayersAttribution`